src/bundle/Resources/views/fieldtypes/preview/content_fields.html.twig

```diff
@@ -208,6 +208,18 @@
             </tr>
         </tbody>
     </table>
+
+    {% if parameters['password_expires_at'] %}
```

Comment:
Now when I look at the usage I feel like this should be more strict, using object and methods rather than string keys.